docs/Getting Started.md

```diff
@@ -22,6 +22,8 @@ host | The computer on which you are editing source code, compiling, and assembl
 target | The platform for which your firmware is built (for example, Raspberry Pi, Raspberry Pi 2, or Beaglebone Black)
 toolchain | The tools required to build code for the target, such as compilers, linkers, binutils, and C runtime
 system | A lean buildroot-based linux distribution that has been customized and cross-compiled for a particular target
+application | The Elixir (and any native, e.g. C) source code for your project. Note that this may entail more than one "application" in Elixir parlance.
```

Comment:
I agree that it's probably a good idea to define "application" in the context of our docs, but I'm thinking maybe we should strictly use this word to refer to an Erlang/Elixir application, to reduce confusion with what is called an application in other languages. This could be a good place to point out the difference, though.